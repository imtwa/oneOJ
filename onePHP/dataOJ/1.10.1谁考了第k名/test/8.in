92 26
5133521 86.2084
3908297 73.1019
332335 36.0338
3490960 43.6679
2342830 88.7271
4974977 79.694
6740276 39.4259
928919 72.0347
5861067 98.5413
3839519 34.5031
6414408 67.8297
2905671 30.8809
6614467 29.8554
3470639 74.3445
683467 42.9799
5553048 64.6454
3131022 63.3669
5927706 62.049
6761286 38.9955
5081641 29.1329
5018237 98.5713
5998230 39.6695
2236097 20.1935
4534398 71.1939
1434823 67.4344
2579198 72.5102
5127333 66.3072
3983846 23.1435
2943961 51.3216
1970092 91.4879
5051110 35.001
3613257 20.3825
2527623 78.4654
5946861 91.8753
684869 63.6152
104138 85.8479
5686808 38.0438
5498891 99.2519
5788080 63.8022
2130778 86.5595
2262502 78.1434
1426509 41.7089
5378609 79.5968
3868991 21.5938
5876398 21.7503
4966372 65.7928
3471354 46.4954
2402466 64.296
223648 41.4321
4304872 39.8528
5765984 37.1636
5967576 81.042
1098456 83.7849
2832752 49.7911
5148885 20.3811
452014 52.4553
3220502 54.4632
1975839 94.7211
6809497 26.2718
4420568 92.8288
2900112 26.0953
3841014 80.2043
1142654 70.9487
4741285 49.8857
4250624 42.3038
5208886 71.7064
6086843 23.8814
2988661 77.5263
3925146 99.7675
1109506 55.5402
3809442 57.7049
2261343 54.4196
912413 61.4932
1389833 77.8112
2942369 22.9669
2218801 29.9204
1044260 99.477
931621 99.5006
609550 73.9134
3773283 42.1426
6132378 40.8215
2430540 89.0097
507769 34.7622
2485289 32.1436
1599341 68.6879
5111597 81.6508
437842 78.4331
514851 82.3263
6875484 23.9179
784673 97.6466
4144502 69.4161
4911186 93.118
