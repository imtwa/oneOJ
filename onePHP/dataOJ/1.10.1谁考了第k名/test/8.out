437842 78.4331